coverage/PENDING_REVIEW/intel/0467-COVERAGE-brw-eu-emit-1366/cov-nested-loops-global-loop-counter-do-while-accumulate-float.amber

```diff
@@ -0,0 +1,660 @@
+#         {
+#             for(int i = _int_0; i < _int_1; i++)
+#             {
+#                 for(int i = _int_0; i < _int_1; i++)
+#                 {
+#                     for(int i = _int_0; i < _int_1; i++)
+#                     {
+#                         for(int i = _int_0; i < _int_1; i++)
+#                         {
+#                             for(int i = _int_0; i < _int_1; i++)
+#                             {
+#                                 for(int i = _int_0; i < _int_1; i++)
+#                                 {
+#                                     for(int i = _int_0; i < _int_1; i++)
+#                                     {
+#                                         for(int i = _int_0; i < _int_1; i++)
+#                                         {
+#                                             for(int i = _int_0; i < _int_1; i++)
+#                                             {
+#                                                 for(int i = _int_0; i < _int_2; i++)
+#                                                 {
+#                                                     for(int i = _int_0; i < _int_1; i++)
+#                                                     {
+#                                                         for(int i = _int_0; i < _int_1; i++)
+#                                                         {
+#                                                             for(int i = _int_0; i < _int_2; i++)
+#                                                             {
+#                                                                 // This inner loop is run four times. The _GLF_global_loop_count
+#                                                                 // changes accordingly: 90, 91, 92, 93.
+#                                                                 do
+#                                                                 {
+#                                                                     _GLF_global_loop_count++;
+#                                                                 }
+#                                                                 while(_GLF_global_loop_count < _GLF_global_loop_bound - _int_10);
+#
+#                                                                 f += _float_1_0;
+#                                                             }
+#                                                         }
+#                                                     }
+#                                                 }
+#                                             }
+#                                         }
+#                                     }
+#                                 }
+#                             }
+#                         }
+#                     }
+#                 }
+#             }
+#         }
+#     }
+#
+#     float sum = _float_0_0;
+#     for(int r = _int_0; _GLF_global_loop_count < _GLF_global_loop_bound; r++)
```

Comment:
Now comment that this loop iterates 7 times, and f is 4.0, so that sum ends up being 4.0*7 == 28.0.